### ISSUE TITLE
Feature Request: Move menu to top level, top of menu

### ISSUE THREAD

**Issue by MightyDrakeC** [gyng/save-in#21]
The previous addon I used had the folders at the top of the image context menu.  I use it enough that having to move down to the bottom and open a sub-menu is really tedious.  I'd like an option to do that here.

Thanks,
Drake Christensen

**Comment by gyng** [comment]
This is really annoying for me too. I tried looking around but I don't see a way to order menus created by extensions. Does anyone know anything about this?

Also see #15

**Comment by Shinkir0** [comment]
I completely agree, I also want it to be so

**Comment by hassanmode** [comment]
You should be able to move it to the top by adding the following to your userChrome.css file. 


/* Move "Save In" menu item to the top */

menu[label="Save In…"] {
    -moz-box-ordinal-group: 0 !important;
}

**Comment by DanaMW** [comment]
The above  didn't  work ...

**Comment by Sjkomy** [comment]
![default](https://user-images.githubusercontent.com/34168091/34447112-9ccf9566-ece9-11e7-91fa-d929d8ad161f.PNG)
So what am i doing wrong then?

**Comment by MightyDrakeC** [comment]
The userChrome.css workaround did not work for me, either.  It did move it up a couple of spots.  But, not to the top.

**Comment by gyng** [comment]
The workaround [above](https://github.com/gyng/save-in/issues/21#issuecomment-349250207) worked for me, moved it right to the top.

There's a guide on how to edit the context menu at https://www.reddit.com/r/firefox/comments/7dvtw0/

**Comment by MightyDrakeC** [comment]
Unfortunately, that page doesn't provide any troubleshooting advice.
![2017-12-30 01_02_00-feature request_ move menu to top level top of menu issue 21 gyng_save-in](https://user-images.githubusercontent.com/12826016/34452117-c5d7c8b6-ecfd-11e7-892a-d1d7b7258d13.png)

**Comment by hassanmode** [comment]
@Sjkomy is this your first time creating a userChrome.css file? 

Either way, make sure that your userChrome.css file is in the right **profile** directory and it's working as it should be. Try doing something simple like changing the color of #urlbar to test to make sure its working.

#urlbar {
background-color: #FF0000 !important;
color: #00FF00 !important; 
}

Also try using the addon's #id selector instead of an attribute selector

#_72d92df5-2aa0-4b06-b807-aa21767545cd__save-in-root {
    -moz-box-ordinal-group: 0 !important;
}

**Comment by Sjkomy** [comment]
> Also try using the addon's #id selector instead of an attribute selector

#_72d92df5-2aa0-4b06-b807-aa21767545cd__save-in-root {
-moz-box-ordinal-group: 0 !important;
}

That worked for me. Thx (and yes never used this before because addons suited me with all i needed, well except Master pass prompt...fucking mozilla)

**Comment by MightyDrakeC** [comment]
Yes, I already had a userChrome.css.  I use Tree Style Tabs, so I had some CSS in there to turn off the stock tabs.  I did double-check with the URL bar color, and it confirmed that I was editing the correct file.

I'm on a different machine than last night, and it's not working at all, here.  I have tried both versions of the CSS.  The "Save in..." is stuck just above Evernote Web Clipper down at the bottom.

Frustratingly, my Browser Toolbox doesn't have the "Four Squares" icon to debug popups.  I'll try it on my other machine later.

**Comment by hassanmode** [comment]
@MightyDrakeC  might have something to do with your user profile. In which case, You could try creating a fresh clean Profile (about:profiles) and install "Save In" there and test the userChrome.css workaround. 

This would be the best way to figure out whats going on.

**Comment by MightyDrakeC** [comment]
That's an idea.  I'll try it later.  Thanks
